scripts/overrides/tei_to_es.rb

```diff
@@ -109,4 +109,17 @@ def citation
     citations
   end
 
+  def date(before=true)
+    if get_list(@xpaths["date"])
+      datestr = get_list(@xpaths["date"]).first
+    else
+      datestr = nil
+    end
+    if get_text(@xpaths["date_not_after"])
+      date_not_after
+    elsif datestr && !datestr.empty?
+      Datura::Helpers.date_standardize(datestr, false)
+    end
+  end
```

Comment:
We don't want to unnecessarily run the code to calculate `datestr` if the first conditional in the second part is going to evaluate true and return `date_not_after`. We should only run the code to try and populate `datestr` after that conditional fails. So I'd rewrite like this:

```ruby
    if get_text(@xpaths["date_not_after"])
      date_not_after
    else
      if get_list(@xpaths["date"])
        datestr = get_list(@xpaths["date"]).first
        Datura::Helpers.date_standardize(datestr, false)
      end
    end
```

This also eliminates the logic of having to check whether datestr is empty, because we know it won't be due to checking `get_list` doesn't return `nil`